src/unix/linux_like/mod.rs

```diff
@@ -982,7 +982,7 @@ pub const PTRACE_EVENT_VFORK_DONE: ::c_int = 5;
 pub const PTRACE_EVENT_EXIT: ::c_int = 6;
 pub const PTRACE_EVENT_SECCOMP: ::c_int = 7;
 // PTRACE_EVENT_STOP was added to glibc in 2.26
-// pub const PTRACE_EVENT_STOP: ::c_int = 128;
+pub const PTRACE_EVENT_STOP: ::c_int = 128;
```

Comment:
Did you check emscripten has this? It seems it doesn't.

Reply:
I've added a `cfg` to disable it on emscripten (their version of `musl` hasn't been updated to have this yet).

Reply:
You should move the item to `/android` and `/linux` instead of using `cfg`.

Reply:
Thanks for the suggestion, I've moved them to the respective files.